packages/components/textarea/src/textarea.ts

```diff
@@ -93,6 +148,40 @@ export class Textarea extends FormControlMixin(HintMixin(LitElement)) {
       }
     }
 
+    if (changes.has('autocomplete')) {
+      if (this.autocomplete) {
+        this.textarea.setAttribute('autocomplete', this.autocomplete);
+      } else {
+        this.textarea.removeAttribute('autocomplete');
+      }
+    }
+
+    if (changes.has('resize')) {
+      if (this.resize) {
+        this.textarea.setAttribute('resize', this.resize);
+      } else {
+        this.resize = 'vertical';
+        this.textarea.setAttribute('resize', this.resize);
+      }
+    }
+
+    if (changes.has('readonly')) {
+      if (this.readonly) {
+        this.textarea.readOnly = this.readonly;
```

Comment:
when you set `this.textarea.readOnly` to `false` it removes the attribute, so this check shouldn't be necessary; 
```
if (changes.has('readonly')) {
    this.textarea.readOnly = this.readonly;
}
```
should work as well